src/DotNet/DotNet.csproj

```diff
@@ -171,6 +174,32 @@
         VersionBand="$(DotNetPreviewVersionBand)"
     />
 
+    <!-- HACK: temporary fixes for iOS and MacCatalyst workloads -->
+    <ReplaceText
+        Input="$(DotNetSdkManifestsDirectory)microsoft.net.sdk.maccatalyst/WorkloadManifest.json"
+        Output="$(DotNetSdkManifestsDirectory)microsoft.net.sdk.maccatalyst/WorkloadManifest.json"
+        OldValue="&quot;description&quot;"
+        NewValue="&quot;extends&quot;: [ &quot;microsoft-net-runtime-maccatalyst&quot; ], &quot;description&quot;"
+    />
```

Comment:
This should hopefully solve this issue on the xamarin-macios side: https://github.com/xamarin/xamarin-macios/pull/12045